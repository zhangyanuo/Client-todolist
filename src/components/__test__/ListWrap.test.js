import ListWrap from '../ListItems';
import React from 'react';
import {shallow} from 'enzyme';

test('render listWrap component',()=>{
    const list ={
        _id:'111',
        name:"aaa",
        isEdit:true,
        status:0
    }
    const wrapper=shallow(
        <ListWrap list={list}/>
    )
    expect(wrapper).toMatchSnapshot();
})