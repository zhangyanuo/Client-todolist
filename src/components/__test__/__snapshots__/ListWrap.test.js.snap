// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render choose component 1`] = `ShallowWrapper {}`;

exports[`render listWrap component 1`] = `ShallowWrapper {}`;
